src/Personal/register.jsx

```diff
@@ -0,0 +1,165 @@
+              leftIconContainerStyle={RegisterView.inputIcon}
+              label="Your Email Address"
+              textContentType="emailAddress"
+              value={email}
+              onChangeText={(value) => setEmail(value)}
+              labelStyle={{ color: 'gray' }}
+            />
+            <Text style={RegisterView.note}>
+              {utils.emailValidation(email) || email === '' ? '' : 'Your email is not correct.' }
+            </Text>
+          </View>
+
+          <View style={RegisterView.inputSection}>
+            <Input
+              placeholder="Password"
+              leftIcon={(
+                <Icon
+                  name="lock"
+                  color="#a8a8a8"
+                />
+              )}
+              leftIconContainerStyle={RegisterView.inputIcon}
+              label="Your password"
+              secureTextEntry={passwordHidden}
+              value={password}
+              onChangeText={(value) => setPassword(value)}
+              labelStyle={{ color: 'gray' }}
+              rightIcon={(
+                <Icon
+                  name={`md-eye${passwordHidden ? '-off' : ''}`}
+                  type="ionicon"
+                  color="gray"
+                  onPress={() => setPasswordHidden(!passwordHidden)}
+                />
+            )}
+            />
+            <Text style={RegisterView.note}>
+              {utils.passwordValidation(password) || password === ''
+                ? '' : 'Password should be at least 8 characters.'}
+            </Text>
+          </View>
+
+          <View style={RegisterView.inputSection}>
+            <Input
+              placeholder="Confirm your password"
+              leftIcon={(
+                <Icon
+                  name="lock"
+                  color="#a8a8a8"
+                />
+              )}
+              leftIconContainerStyle={RegisterView.inputIcon}
+              label="Confirm your password"
+              secureTextEntry={confirmPasswordHidden}
+              value={confirmPassword}
+              onChangeText={(value) => setConfirmPassword(value)}
+              labelStyle={{ color: 'gray' }}
+              rightIcon={(
+                <Icon
+                  name={`md-eye${confirmPasswordHidden ? '-off' : ''}`}
+                  type="ionicon"
+                  color="gray"
+                  onPress={() => setConfirmPasswordHidden(!confirmPasswordHidden)}
+                />
+            )}
+            />
+            <Text style={RegisterView.note}>
+              {password === confirmPassword || confirmPassword === '' ? '' : 'Password is not the same.'}
+            </Text>
+          </View>
+          {
+             (errorMsgDisplay) ? (
+               <View style={[RegisterView.inputSection, RegisterView.errorBox]}>
+                 <Text style={RegisterView.errorText}>{user.message}</Text>
+               </View>
+             ) : (null)
+          }
+          <View>
+            <SubmitBtn
```

Comment:
we should indicate somehow the signup process so the user will know whats going on under the hood